terraform/elasticsearch.tf.json.template.py

```diff
@@ -65,19 +66,6 @@
                                 },
                                 "Action": "es:*",
                                 "Resource": "arn:aws:es:${local.region}:${local.account_id}:domain/" + domain + "/*"
-                            },
-                            {
-                                "Effect": "Allow",
-                                "Principal": {
-                                    "AWS": "*"
-                                },
-                                "Action": "es:*",
-                                "Resource": "arn:aws:es:${local.region}:${local.account_id}:domain/" + domain + "/*",
-                                "Condition": {
-                                    "IpAddress": {
-                                        "aws:SourceIp": []
-                                    }
-                                }
```

Comment:
This hunk should be a separate commit titled "Remove obsolete IAM policy section allowing access to ES by IP"